code/backend/batch/BatchPushResults.py

```diff
@@ -7,14 +7,14 @@
 from utilities.helpers.azure_blob_storage_client import AzureBlobStorageClient
 from utilities.helpers.env_helper import EnvHelper
 from utilities.helpers.embedders.embedder_factory import EmbedderFactory
+from utilities.search.Search import Search
 
 bp_batch_push_results = func.Blueprint()
 logger = logging.getLogger(__name__)
 logger.setLevel(level=os.environ.get("LOGLEVEL", "INFO").upper())
 
 
-def _get_file_name_from_message(msg: func.QueueMessage) -> str:
-    message_body = json.loads(msg.get_body().decode("utf-8"))
+def _get_file_name_from_message(message_body: any) -> str:
```

Comment:
My editor is complaining that these types should be `Any`?

Reply:
str | byte  ?

Reply:
`Any` would require an import from `typings`: 

![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/2767292/6ce0a6c2-3402-4306-a1bb-15e352981d9e)

VSCode, Pycharm, and Lance are all happy with `any` for @gaurarpit and I

Reply:
It thinks it's the `any()` function

Reply:
updated now following discussion with @cecheta